cylc/flow/workflow_files.py

```diff
@@ -1226,6 +1226,8 @@ def check_deprecation(path, warn=True):
         cylc.flow.flags.cylc7_back_compat = True
         if warn:
             LOG.warning(SUITERC_DEPR_MSG)
+        return True
+    return False
```

Comment:
Can't see how these returns are being used. I'm probably missing something though.

Reply:
@`validate_reinstall.py::147`

Reply:
Hmm, maybe you have not pushed changes? I have that line as checking... `if not workflow_running:` and I have this as adding returns to `check_deprecation`. I have pulled changes and nothing coming through for me.